### PR TITLE
Update CSS token per new design system theming capability

### PR DESCRIPTION
**Pull request by fajar-apri-alaska**
# Alaska Airlines Pull Request

**Fixes:** #163 

## Summary:

- Update the design tokens to reflect the new theming conventions regarding tokens from the AuroDesignTokens and JetstreamDesignTokens repo.

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by fajar-apri-alaska** [comment]
The  `npm ci` is failing because some auro components have conflicting peer dependency for `@aurodesignsystem/webcorestylesheets` package. Should I hold of this until those components are updated?

**Comment by fajar-apri-alaska** [comment]
Also bumping `auro-dropdown` version to 2.9.9 as a temporary solution for this PR, because somehow the run test are failing from the `auro-dropdown` version in main branch (2.9.6)

File: src/style.scss [hunk 1]
```diff
@@ -19,13 +23,12 @@
...
-      /* 2.5rem if --auro-size-500 not defined */
-      min-height: var(--auro-size-500, 2.5rem);
-      max-height: var(--auro-size-500, 2.5rem);
+      min-height: var(--ds-size-500, $ds-size-500);
+      max-height: var(--ds-size-500, $ds-size-500);
 
       /* stylelint-disable */
       &:hover {
-        background-color: var(--auro-color-ui-bkg-hover-on-light);
+        background-color: var(--ds-color-ui-bkg-hover-default);
```
Comment (jason-capsule42) [inline src/style.scss:31]: Add the fallback value

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: src/style.scss (hunk 1 above)
Comment (fajar-apri-alaska) [reply src/style.scss:31]: My bad, thanks for noticing.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**